spec/lib/augeas_spec/fixtures.rb

```diff
@@ -43,8 +43,8 @@ def apply!(*resources)
     loglevels = Puppet::Util::Log.levels[2, 999]
     againlogs = @logs.select { |log| loglevels.include? log.level }
 
-    againlogs.should eq([]), "expected no change on second run (idempotence check),\n     got: #{againlogs.inspect}"
-    txn_idempotent.any_failed?.should_not eq(true), 'expected no change on second run (idempotence check), got a resource failure'
+    expect(againlogs).to eq([]), 'expected no change on second run (idempotence check)'
+    expect(txn_idempotent.any_failed?).to be_false, 'expected no change on second run (idempotence check), got a resource failure'
```

Comment:
The check should be comparing to `be_nil`
```suggestion
    expect(txn_idempotent.any_failed?).to be_nil, 'expected no change on second run (idempotence check), got a resource failure'
```